engine/clients/opensearch/upload.py

```diff
@@ -56,16 +63,34 @@ def upload_batch(cls, batch: List[Record]):
             index=OPENSEARCH_INDEX,
             body=operations,
             params={
-                "timeout": 300,
+                "timeout": OPENSEARCH_BULK_INDEX_TIMEOUT,
             },
         )
 
     @classmethod
+    @backoff.on_exception(
+        backoff.expo,
+        TransportError,
+        max_time=OPENSEARCH_FULL_INDEX_TIMEOUT,
+        on_backoff=_index_backoff_handler,
+    )
     def post_upload(cls, _distance):
-        cls.client.indices.forcemerge(
+        print(
+            "Updated the index settings back to the default and waiting for indexing to be completed."
+        )
+        # Update the index settings back to the default
+        refresh_interval = "1s"
+        cls.client.indices.put_settings(
             index=OPENSEARCH_INDEX,
-            params={
-                "timeout": 300,
-            },
+            body={"index": {"refresh_interval": refresh_interval}},
```

Comment:
Is `cls.client.indices.refresh(index=OPENSEARCH_INDEX)` better?

Reply:
i believe it's best as is, meaning:
- we disable refresh during indexing
- we enable it back after it